configs/rotated_retinanet/README.md

```diff
@@ -3,7 +3,9 @@
 <!-- [ALGORITHM] -->
 ## Abstract
 
-![illustration](https://raw.githubusercontent.com/zytx121/image-host/main/imgs/retina.png)
+<div align=center>
+<img src="https://raw.githubusercontent.com/zytx121/image-host/main/imgs/retina.png" width="400"/>
+</div>
 
 The highest accuracy object detectors to date are based on a two-stage approach popularized by R-CNN, where a classifier is applied to a sparse set of candidate object locations. In contrast, one-stage detectors that are applied over a regular, dense sampling of possible object locations have the potential to be faster and simpler, but have trailed the accuracy of two-stage detectors thus far. In this paper, we investigate why this is the case. We discover that the extreme foreground-background class imbalance encountered during training of dense detectors is the central cause. We propose to address this class imbalance by reshaping the standard cross entropy loss such that it down-weights the loss assigned to well-classified examples. Our novel Focal Loss focuses training on a sparse set of hard examples and prevents the vast number of easy negatives from overwhelming the detector during training. To evaluate the effectiveness of our loss, we design and train a simple dense detector we call RotatedRetinaNet. Our results show that when trained with the focal loss, RetinaNet is able to match the speed of previous one-stage detectors while surpassing the accuracy of all existing state-of-the-art two-stage detectors.
```

Comment:
These words are somewhat misleading. 
The logic starts from the comparison between 1-stage and 2-stage detectors, then reveals the cause of the performance gap between them, and finally goes to focal loss and describes how it works. The logic goes smoothly until here and matches perfectly with the work of RetinaNet.
But just before the last sentence we suddenly shift to RotatedRetinaNet (I would rather believe that we shall use RetinaNet here to make the logic consistent)
Meanwhile, shall we add more descriptions here discussing what we do to the original RetinaNet so that we make it a rotated object detector? Since the Retina-H and Retina-R are the baselines of various rotated object detectors, some in-depth discussion would be beneficial to the community.